### ISSUE TITLE
Event Handler and Authorization

### ISSUE THREAD

**Issue by eudyptula** [arrowhead-f/core-java#26]
Shouldn't the event handler verify subscribers and publishers with the authorisation system? If I'm not mistaken, everything can send and receive anything in the current state...

**Comment by hegeduscs** [comment]
Yes, the EH should handle the authorization tokens, as any other service. However, the integration with the Auth and/or token schema might kill scalability. 

It is not implemented yet, it may be worth visiting this conceptual issue. It should be, in my opinion.

**Comment by uzoltan** [comment]
This would not be the bottleneck on the already existing scalability issues on the event handler.
